src/coreclr/vm/comsynchronizable.cpp

```diff
@@ -1127,17 +1116,3 @@ extern "C" void QCALLTYPE ThreadNative_ResetAbort()
         pThread->UnmarkThreadForAbort(EEPolicy::TA_Safe);
     }
 }
-
-FCIMPL0(INT32, ThreadNative::GetCurrentProcessorNumber)
-{
-    FCALL_CONTRACT;
-
-#ifndef TARGET_UNIX
-    PROCESSOR_NUMBER proc_no_cpu_group;
-    GetCurrentProcessorNumberEx(&proc_no_cpu_group);
-    return (proc_no_cpu_group.Group << 6) | proc_no_cpu_group.Number;
-#else
-    return ::GetCurrentProcessorNumber();
```

Comment:
Delete `GetCurrentProcessorNumber` and `PAL_HasGetCurrentProcessorNumber` from CoreCLR PAL?

Reply:
I thought they are used by GC, but GC appears to have its own copy.

Reply:
Deleted